mmcv/ops/csrc/pytorch/cuda/cudabind.cpp

```diff
@@ -1867,3 +1867,46 @@ REGISTER_DEVICE_IMPL(prroi_pool_forward_impl, CUDA, prroi_pool_forward_cuda);
 REGISTER_DEVICE_IMPL(prroi_pool_backward_impl, CUDA, prroi_pool_backward_cuda);
 REGISTER_DEVICE_IMPL(prroi_pool_coor_backward_impl, CUDA,
                      prroi_pool_coor_backward_cuda);
+
+
+void BezierAlignForwardCUDAKernelLauncher(Tensor input, Tensor rois, Tensor output,
+                                       int aligned_height, int aligned_width,
+                                       float spatial_scale, int sampling_ratio,
+                                        bool aligned);
+
+void BezierAlignBackwardCUDAKernelLauncher(Tensor grad_output, Tensor rois,
+                                        Tensor grad_input, int aligned_height,
+                                        int aligned_width, float spatial_scale,
+                                        int sampling_ratio,
+                                        bool aligned);
+
+void bezier_align_forward_cuda(Tensor input, Tensor rois, Tensor output,
```

Comment:
`BezierAlignForwardCUDAKernelLauncher` and `bezier_align_forward_cuda` share the same arguments. We do not need to wrap it again. Just register it with:
`REGISTER_DEVICE_IMPL(bezier_align_forward_impl, CUDA, BezierAlignForwardCUDAKernelLauncher);`